### ISSUE TITLE
Something wrong with date-fns

### ISSUE THREAD

**Issue by nkt** [darrenscerri/duplicate-package-checker-webpack-plugin#7]
Hello, thanks for this project!
I've got following warning on my projects:
```
WARNING in duplicate-package-checker:
  <undefined>
    undefined ./~/date-fns/parse
    undefined ./~/date-fns/compare_asc
    undefined ./~/date-fns/compare_desc
    undefined ./~/date-fns/difference_in_calendar_months
    undefined ./~/date-fns/difference_in_milliseconds
    undefined ./~/date-fns/difference_in_months
    undefined ./~/date-fns/difference_in_seconds
    undefined ./~/date-fns/distance_in_words
    undefined ./~/date-fns/distance_in_words_to_now
    undefined ./~/date-fns/is_date
    undefined ./~/date-fns/locale/_lib
    undefined ./~/date-fns/locale/en
```

**Comment by darrenscerri** [comment]
Can you provide more information on how `date-fns` is being required?

**Comment by darrenscerri** [comment]
Is it possible that what's requiring `date-fns` has a `package.json` without the `name` property?

**Comment by darrenscerri** [comment]
Issue fixed with v1.2.2. Issue was do to non-root `package.json` files in `date-fns`.

https://github.com/date-fns/date-fns/issues/264#issuecomment-265128399
https://github.com/jsdnxx/find-root/issues/2

**Comment by nkt** [comment]
Thanks!
